### PR TITLE
Make CSS easier to read

### PR DESCRIPTION
**Pull request by starcalibre**
Added a few things to make the CSS a little easier to read:

* Add comments to better separate between sections
* Add comments to explain some of the less obvious parts
* More consistency with colour selections and formatting

Also a small change to the scatterplot function..

* Use CSS to style scatterplot points where possible, instead of using the D3 selector.

@jni - Not sure if you'll be notified of this so pinging you just in case :)

### CODE REVIEW

File: public/css/style.css [hunk 1]
```diff
@@ -1,4 +1,5 @@
-/* Main Page CSS */
+/* MAIN PAGE CSS */
```
Comment (jni) [inline public/css/style.css]: WHY ARE YOU YELLING

File: public/css/style.css [hunk 2]
```diff
@@ -17,26 +19,39 @@
...
+    display: inline-block;
+    float: none;
+    text-align: left;
+    margin-right: -4px;
 }
 
+/* Centered rows for grid display in Nebula */
 .row-centered {
-    text-align:center;
+    text-align :center;
```
Comment (jni) [inline public/css/style.css]: Why put a leading instead of trailing space? Guessing this is a typo... =)

File: public/css/style.css [hunk 3]
```diff
@@ -17,26 +19,39 @@
...
 .plotbox {
     text-align: center;
 }
 
+/* GENERAL PLOT CSS */
+
+.axis path, .axis line {
+    fill: none;
+    stroke: black;
+    shape-rendering: crispEdges;
```
Comment (jni) [inline public/css/style.css]: Haha this is great value. I think we might want to go with superCrisp though. =P

**Comment by jni** [comment]
Very clean! Just one very minor comment. =)

File: public/css/style.css (hunk 2 above)
Comment (starcalibre) [inline public/css/style.css]: That's a typo :P Thanks

**Comment by starcalibre** [comment]
@jni this should be good to merge :)

**Comment by jni** [comment]
So it is. =)

**Pull request merged**